app/frontend/src/submissions/components/SubmissionPreview/PreviewMap.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
You're going to need to pull in the esri map layers to be consistent with maps on submission and well detail.